working-groups/WORKING-GROUPS.md

```diff
@@ -53,17 +53,22 @@ other system (Apple Calendar or Outlook, for example),
 
 The current working groups are:
 
-- [Serving API](#serving-api)
-- [Client](#client)
-- [Documentation + User Experience](#documentation--user-experience)
-- [Eventing](#eventing)
-- [Eventing Kafka](#eventing-kafka)
-- [Eventing Sources](#eventing-sources)
-- [Networking](#networking)
-- [Operations](#operations)
-- [Productivity](#productivity)
-- [Scaling](#scaling)
-- [Security](#security)
+- [Working Groups](#working-groups)
```

Comment:
My vscode automatically formatted this.. By checking https://github.com/knative/community/blob/b13ee779c01bad5b30e40444e4ec87f74ee7a9a4/working-groups/WORKING-GROUPS.md it seems to be a desirable change though.